src/views/add-member.html

```diff
@@ -8,7 +8,7 @@
   <h1 class="govuk-heading-l">Add a GitHub Member</h1>
 
   <p class="govuk-body">
-    When a user requires access to our GitHub organisations (e.g they have joined The Cabinet Office) they should be added to the cabinetoffice GitHub organisation.
+    When a user requires access to our GitHub organisations (e.g they have joined {{ ORG_NAME }}) they should be added to the {{ ORG_NAME }} GitHub organisation.
```

Comment:
Think we should have 2 global vars, one for department name, one for GitHub organisation name. In our case, Cabinet Office has a lot of GitHub organisations for example so could use the app across multiple orgs.